src/taskpane/outlook.ts

```diff
@@ -15,6 +15,6 @@ const insertText = async (text: string) => {
   } catch (error) {
     console.log("Error: " + error);
   }
-};
+}
 
 export default insertText;
```

Comment:
You kept this here but removed it in other cases . . . we should be consistent

Reply:
done